pandas/tests/tseries/offsets/test_ticks.py

```diff
@@ -267,9 +267,15 @@ def test_tick_rdiv(cls):
     delta = off.delta
     td64 = delta.to_timedelta64()
 
-    with pytest.raises(TypeError):
+    msg = (
+        "unsupported operand type\\(s\\) for \\/: 'int'|'float' and "
+        f"'{'.'.join([cls.__module__, cls.__name__])}'"
```

Comment:
can you do this join outside of the fstring

Reply:
As suggested i have changed it